src/main/resources/graphql/service/mutation.graphqls

```diff
@@ -153,6 +182,210 @@ input MultipleChoiceAnswerInput {
     feedback: String @Size(max: 1000)
```

Comment:
@myluki2000 should I already convert every string (where it makes sense) to resource markdown?
I don't know if we want resource markdown everywhere or maybe somewhere just "normal" markdown

Reply:
I think it makes sense to do it from the start. Also I can't come up with any situation where it would make sense to only allow normal markdown instead of ResourceMarkdown